Doc/whatsnew/3.14.rst

```diff
@@ -92,6 +93,75 @@ If you encounter :exc:`NameError`\s or pickling errors coming out of
 New features
 ============
 
+.. _whatsnew314-pep750:
+
+PEP 750: Template Strings
+-------------------------
+
+Template string literals (t-strings) are a generalization of f-strings,
+using a ``t`` in place of the ``f`` prefix. Instead of evaluating
+to ``str``, t-strings evaluate to a new ``string.templatelib.Template`` type:
```

Comment:
```suggestion
to :class:`str`, t-strings evaluate to a new :class:`string.templatelib.Template` type:
```

Reply:
The cross-reference to `Template` will fail until it's documented, so add a `!` to suppress it (and not fail the build). As Hugo said we can add/improve docs post implementation merge / after beta freeze.
```suggestion
to :class:`str`, t-strings evaluate to a new :class:`!string.templatelib.Template` type:
```